### PR TITLE
fix(internal): Misc fixes and features for commands

### PR DESCRIPTION
**Pull request by craciunoiuc**
### Prerequisite checklist

  - [ ] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [ ] Tested your changes against relevant architectures and platforms;
  - [ ] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### CODE REVIEW

File: internal/cli/kraft/cloud/instance/create/create.go [hunk 1]
```diff
@@ -454,7 +454,12 @@ func Rollout(ctx context.Context, opts *CreateOptions, newInstance *kcinstances.
 			req.MemoryMB = &opts.Memory
 		}
 
-		if _, err := opts.Client.Instances().WithMetro(opts.Metro).Create(ctx, req); err != nil {
+		cresp, err := opts.Client.Instances().WithMetro(opts.Metro).Create(ctx, req)
+		if err != nil {
+			return fmt.Errorf("could not create a new instance: %w", err)
+		}
+		_, err = cresp.FirstOrErr()
+		if err != nil {
```
Comment (antoineco) [inline internal/cli/kraft/cloud/instance/create/create.go]:
```suggestion
		if _, err = cresp.FirstOrErr(); err != nil {
```

File: internal/cli/kraft/cloud/scale/add/add.go [hunk 2]
```diff
@@ -189,7 +189,12 @@ func (opts *AddOptions) Run(ctx context.Context, args []string) error {
 		stepPol.Steps = append(stepPol.Steps, s)
 	}
 
-	if _, err = opts.Client.Autoscale().WithMetro(opts.Metro).AddPolicy(ctx, id, stepPol); err != nil {
+	addPolicyResp, err := opts.Client.Autoscale().WithMetro(opts.Metro).AddPolicy(ctx, id, stepPol)
+	if err != nil {
+		return fmt.Errorf("could not add configuration: %w", err)
+	}
+	_, err = addPolicyResp.AllOrErr()
+	if err != nil {
```
Comment (antoineco) [inline internal/cli/kraft/cloud/scale/add/add.go]:
```suggestion
	if _, err = addPolicyResp.AllOrErr(); err != nil {
```

File: internal/cli/kraft/cloud/scale/initialize/initialize.go [hunk 3]
```diff
@@ -201,9 +201,15 @@ func (opts *InitOptions) Run(ctx context.Context, args []string) error {
 		req.CooldownTimeMs = &cooldownTimeMs
 	}
 
-	if _, err = opts.Client.Autoscale().WithMetro(opts.Metro).CreateConfiguration(ctx, req); err != nil {
+	scaleResp, err := opts.Client.Autoscale().WithMetro(opts.Metro).CreateConfiguration(ctx, req)
+	if err != nil {
 		return fmt.Errorf("could not create configuration: %w", err)
 	}
+	_, err = scaleResp.AllOrErr()
+	if err != nil {
```
Comment (antoineco) [inline internal/cli/kraft/cloud/scale/initialize/initialize.go]:
```suggestion
	if _, err = scaleResp.AllOrErr(); err != nil {
```

**Review by antoineco — APPROVED** [review]
Approving with minor suggestions for style consistency.

File: internal/cli/kraft/cloud/utils/print.go [hunk 4]
```diff
@@ -600,6 +600,70 @@ func PrintLimits(ctx context.Context, format string, resp *kcclient.ServiceRespo
 	return table.Render(iostreams.G(ctx).Out)
 }
 
+func PrintFeatures(ctx context.Context, format string, resp *kcclient.ServiceResponse[kcusers.QuotasResponseItem]) error {
```
Comment (antoineco) [inline internal/cli/kraft/cloud/utils/print.go]: Maybe `PrintQuotaFeatures` to provide better context?

Comment (craciunoiuc) [reply internal/cli/kraft/cloud/utils/print.go]: same for `PrintLimits` I guess then

**Pull request merged**